src/linear_feedback_controller.cpp

```diff
@@ -483,6 +488,9 @@ void LinearFeedbackController::acquireSensorAndPublish(
     eigen_sensor_msg_.joint_state.effort(i) =
         getJointMeasuredTorque(pin_to_hwi_[i]) -
         in_torque_offsets_[pin_to_hwi_[i]];
+    if (std::isnan(eigen_sensor_msg_.joint_state.effort(i))) {
```

Comment:
So here we should keep sending NaNs?

Reply:
I do not see why we should send NaNs here.

Reply:
The question I was asking is, should we remove the NaNs from the measures if there are any?
In my opinion we should either send a value that has been measured/estimated or send a NaN (and not just replace a NaN by 0).

My question was mainly to know if sending NaNs could lead to unwanted behavior in the computation of the MPC (in that case we should change them to 0).
There might be some NaNs in the effort measurement when the wrist are controlled because there are no torque sensors on those joints. In my case I know it won't change anything because I don't use the effort measurements at all.